docs/developer/available-plugins/stripe.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Stripe
+---
+
+import Chart from "@site/components/Chart";
+
+Stripe plugin uses [custom payment flow](https://stripe.com/docs/payments/accept-a-payment?platform=web&ui=elements) to process customer transactions.
```

Comment:
Would like to see introduction here, what kind of requirements are needed for this app to operate:
- private API key?
- how configuration should be made?

Reply:
@krzysztofwolski  Maybe, it would be good idea to add a section about configuration in dashboard docs?

Reply:
Good idea! Can you do that?

Reply:
Yes, I can